### ISSUE TITLE
The countries and cities on the map are still in English

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1368]
On the map itself there is Cologne, Germany and so on. This should be in the language the map is actually shown.

**Comment by trugwaldsaenger** [comment]
Good point. I saw this, too, but did not think of it as an issue. In my opinion not very high priority, but we should take it to the backlog.

**Comment by karindr** [comment]
The coutries are now displayed in German too, that's great. But for example not for Portuguese, where we have an almost complete transaltion.
